<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Administration</class>
 <widget class="QDialog" name="Administration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администрирование</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 127);</string>
  </property>
  <widget class="QLineEdit" name="AddObjectLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveStudentButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Удалить студента</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveObjectButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Удалить предмет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveTeacherButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Удалить преподавателя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddObjectButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Добавить предмет</string>
   </property>
  </widget>
  <widget class="QComboBox" name="RemoveObjectComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AssignObject">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Назначить предмет группе</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>881</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="GroupLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="GroupButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Создать группу</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>291</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="StudentGroupComboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="StudentSurnameComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="StudentNameComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>291</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QComboBox" name="TeacherObjectComboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="TeacherSurnameComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="TeacherNameComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>261</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QComboBox" name="GroupComboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="ObjectComboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
